snafu/image_pull_wrapper/image_pull_wrapper.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+import os
+import argparse
+from .trigger_image_pull import Trigger_image_pull
+
+
+class image_pull_wrapper:
```

Comment:
(nitpick) Not the most Pythonic class name (snake_case vs. PascalCase) but I do see that's what is being used by other tests.

Reply:
yeah, lots of class names can be updated... if we set a standard we should stick to it.